libs/feeds_processing/src/adfs_gen_calldata.rs

```diff
@@ -48,7 +47,11 @@ pub async fn adfs_serialize_updates(
     net: &str,
     feed_updates: &BatchedAggegratesToSend,
     feeds_metrics: Option<Arc<RwLock<FeedsMetrics>>>,
-    feeds_config: Arc<RwLock<HashMap<u32, FeedConfig>>>,
+    feeds_config: HashMap<u32, FeedStrideAndDecimals>,
```

Comment:
Perhaps rename parameter to `strides_and_decimals`, as it's called elsewhere. `feeds_config` is misleading, since there's a type with the same name.